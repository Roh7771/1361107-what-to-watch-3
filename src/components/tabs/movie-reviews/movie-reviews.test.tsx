import * as React from "react";
import * as renderer from "react-test-renderer";
import MovieReviews from "./movie-reviews";

it(`<MovieReviews /> should render correctly`, () => {
  const tree = renderer
    .create(<MovieReviews filmComments={[]}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
