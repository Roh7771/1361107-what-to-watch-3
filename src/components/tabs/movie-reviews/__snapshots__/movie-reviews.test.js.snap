// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieReviews /> should render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  >
    <div
      className="review"
    >
      <blockquote
        className="review__quote"
      >
        <p
          className="review__text"
        >
          Some comment
        </p>
        <footer
          className="review__details"
        >
          <cite
            className="review__author"
          >
            Some User
          </cite>
          <time
            className="review__date"
            dateTime="Sun Mar 15 2020"
          >
            March 15, 2020
          </time>
        </footer>
      </blockquote>
      <div
        className="review__rating"
      >
        9
      </div>
    </div>
  </div>
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
